adam-core/src/main/scala/org/bdgenomics/adam/rdd/variant/GenotypeRDD.scala

```diff
@@ -159,17 +162,39 @@ case class GenotypeRDD(rdd: RDD[Genotype],
   /**
    * Explicitly saves to VCF.
    *
+   * @param args Arguments defining where to save the file.
+   * @param stringency The validation stringency to use when writing the VCF.
+   *   Defaults to LENIENT.
+   */
+  def saveAsVcf(args: ADAMSaveAnyArgs,
```

Comment:
I wouldn't have added it if I thought it should be removed ;)

As mentioned elsewhere, save methods are a bit of a mess and need to be made consistent across subclasses of `GenomicRDD`.  I'm ok if this PR makes it a little worse before it gets better.